src/Components/Search/Search.js

```diff
@@ -1,17 +1,51 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { getListName } from '../../actions';
+import { getBooks } from '../../apiCalls.js';
 import './Search.css';
 
 class Search extends Component {
-  constructor() {
-    super();
-    this.state = {}
+  constructor(props) {
+    super(props);
+  }
+
+  getSearchList = (event) => {
+    this.props.getListName(this.state.listName)
+  }
+
+  updateValue = (event) => {
+    this.setState({listName: event.target.value})
   }
 
   render() {
     return (
-      <h3>Search goes here</h3>
+      <div className='search'>
+        <select
+        className='search-menu'
+        name='list-genre'
+        onChange={this.updateValue}
```

Comment:
This drop-down is a nice, user-friendly way of controlling this input, figuring out that the target was defaulting to the button instead of the input field was a good catch